### DISCUSSION TITLE
Radiostream with Songtitle and Artist ? Rockantenne

### DISCUSSION THREAD

**Discussion by ds2k5** [ampache/ampache#4043] [General]
Hi,
is it possible to get Artist an Songtitle in Radiostream ?

https://stream.rockantenne.bayern/heavy-metal/stream/mp3

https://www.rockantenne.bayern/rockhoeren/streams/heavy-metal

Thanks

**Comment by lachlan-00** [comment]
The webplayer loads on URL load so a stream doesn't refresh until the stream is updated/next track

I'm sure it's probably easy enough to do in ajax/JS but not something i've looked at doing yet
